services/export/src/utils/stream-results.js

```diff
@@ -29,7 +29,13 @@ const executor = async (args) => {
   });
   const nodes = getAsArray(data, 'edges').map((edge) => {
     const { node } = edge;
-    const row = fields.reduce((o, field) => ({ ...o, [field]: node[field] }), {});
+
+    const reducer = (o, field) => {
```

Comment:
If the intention is to convert null and undefined fields to an empty string, this should be sufficient
```js
const row = fields.reduce((o, field) => ({ ...o, [field]: node[field] == null ? '' : node[field] }), {});
```

Reply:
Ah cool, I didn't know use ternary conditional right there in the inline function.  Thanks, made that change.